community/samples/serving/helloworld-java-quarkus/README.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "Hello World - Java (Quarkus)"
+linkTitle: "Java"
+weight: 1
+type: "docs"
+---
+
+{{% readfile file="README.md" relative="true" markdown="true" %}}
+
+A simple [JAX-RS REST API](https://github.com/jax-rs) application that is written in Java and uses [QuarkusIO](https://quarkus.io/).
+
+It uses an ENV variable `MESSAGE_PREFIX`(defaults to value `Hello`) and prints a message of format `${MESSAGE_PREFIX} Knative World!`.
+
+## Before you begin
+
+You must meet the following requirements to run this sample:
+
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/docs/install/README.md)
+  if you need to create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
```

Comment:
Reordering steps to highlight Knative Build as optional (and that Docker is required to push to Docker Hub):
```suggestion
- You must install [Docker](https://www.docker.com) and have a Docker Hub account to which 
   you are able to upload your sample's container image.
- You can choose to use Knative Build or use Docker to build your sample's container image.
   - If you choose to use the [Knative Build component](../../../../docs/build/installing-build-component.md), 
     you must ensure that the Knative Build component is also installed in the cluster where you installed 
     Knative Serving. This method uses the [buildah](https://buildah.io) tool.
```

Reply:
removed knative build section.